Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Chip.h

```diff
@@ -146,6 +146,7 @@ class Chip : public TObject
     /// @param p Point to be added
     void InsertPoint(Point *p);
```

Comment:
Hi Sandro,

No, in this particular place the "const" will probably not work. TObjArray::AddLast() will not allow.
Otherwise, in general... I am sure there are many similar places that would call for such a fix. We just cannot fix them all at once...

Cheers,
Iouri.